src/collective/solr/tests/test_server.py

```diff
@@ -572,12 +572,6 @@ def testReindexPathIndex(self):
         ob._setId('new_id')
         commit()
 
-        # No change in solr so far
```

Comment:
@gforcada can you elaborate why you removed those lines?

Reply:
As explained on the commit I made removing those lines:

```
The last version of collective.indexing (2.1) makes that part of the
test redundant. As soon as a transaction.commit happen, solr gets
notified.
```